src/main.py

```diff
@@ -58,13 +61,17 @@ def execute_block(block_name):
         return 'State must be provided when resuming', HTTPStatus.BAD_REQUEST
 
     # Create new dictionary that will only contain parameters included in block specification
```

Comment:
I think it would be nice to extract lines 63 - 83 to a separate function clean_params. This would create a nice, testable method.

Reply:
I think also extracting a function get_requested_params is useful. Some unit tests on that would help tremedously in communicating what exactly is meant by the different terms.